packages/clerk-js/src/ui/components/SignIn/__tests__/SignInStart.test.tsx

```diff
@@ -199,7 +199,7 @@ describe('SignInStart', () => {
       expect(fixtures.signIn.create).toHaveBeenCalled();
       expect(fixtures.signIn.authenticateWithRedirect).toHaveBeenCalledWith({
         strategy: 'saml',
-        redirectUrl: 'http://localhost/#/sso-callback?redirect_url=http%3A%2F%2Flocalhost%2F',
+        redirectUrl: 'http://localhost/sso-callback',
```

Comment:
This is probably a mistake as the sso-callback needs to have a redirectURL passed-in.. investigating